samza-kv/src/main/scala/org/apache/samza/storage/kv/SerializedKeyValueStoreMetrics.scala

```diff
@@ -36,6 +35,10 @@ class SerializedKeyValueStoreMetrics(
   val bytesSerialized = newCounter("bytes-serialized")
   val bytesDeserialized = newCounter("bytes-deserialized")
   val maxRecordSizeBytes = newGauge("max-record-size-bytes", 0L)
+  val record_key_size_percentiles = java.util.Arrays.asList[java.lang.Double](10D, 50D, 90D, 99D)
+  val record_value_size_percentiles = java.util.Arrays.asList[java.lang.Double](10D, 50D, 90D, 99D)
+  val recordKeySizeBytes = newHistogram("key-size-bytes-histogram",record_key_size_percentiles )
+  val recordValueSizeBytes = newHistogram("value-size-bytes-histogram", record_value_size_percentiles)
```

Comment:
Maybe add javadocs to explain how this works, the histogram stores counts for each percentile range? 
and the count for each range is emitted as a gauge?